indico/modules/users/templates/personal_data.html

```diff
@@ -9,6 +9,27 @@
                 <div class="i-box-title">
                     {%- trans %}Details{% endtrans -%}
                 </div>
+                <div class="i-box-buttons">
+                    {% if not user.is_blocked %}
+                        <button class="ui negative small button"
+                                data-href="{{ url_for('.user_block') }}"
+                                data-method="PUT"
+                                data-confirm="This user will no longer have access to Indico.<br>
+                                Are you sure you want to revoke his access?"
+                                data-reload-after>
+                            {% trans %}Block Access{% endtrans %}
```

Comment:
"Block user"?

Reply:
I had "Block user" but changed it because *blocking a user* could be confused with you blocking a person rather than blocking the person's access to the website.

Reply:
but isn't that the same?

Reply:
We agreed to keep "Block User" since the opposite, a feature to just block someone's activity, won't be considered.